kde-apps/kompare/kompare-25.04.0.recipe

```diff
@@ -0,0 +1,137 @@
+  * graphical interface to commonly used diff command line options
+  * switch source and destination with one command
+  * diff statistics"
+HOMEPAGE="https://apps.kde.org/kompare/"
+COPYRIGHT="2010-2025 KDE Organisation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/kompare-$portVersion.tar.xz"
+CHECKSUM_SHA256="6cede0023d7e8ca55690cb302f13050649b57dbbaa5ea5410059ac0cacf8121e"
+ADDITIONAL_FILES="kompare.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	kompare$secondaryArchSuffix = $portVersion
+	app:Kompare$commandSuffix = $portVersion
+	lib:libkomparedialogpages$secondaryArchSuffix = 6
+	lib:libkompareinterface$secondaryArchSuffix = 6
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libkomparediff2$secondaryArchSuffix
+	# KF6
+	lib:libKF6Codecs$secondaryArchSuffix
+	lib:libKF6ColorScheme$secondaryArchSuffix
+	lib:libKF6Completion$secondaryArchSuffix
+	lib:libKF6ConfigCore$secondaryArchSuffix
+	lib:libKF6ConfigWidgets$secondaryArchSuffix
+	lib:libKF6CoreAddons$secondaryArchSuffix
+	lib:libKF6GuiAddons$secondaryArchSuffix
+	lib:libKF6I18n$secondaryArchSuffix
+	lib:libKF6JobWidgets$secondaryArchSuffix
+	lib:libKF6KIOCore$secondaryArchSuffix
+	lib:libKF6Parts$secondaryArchSuffix
+	lib:libKF6Service$secondaryArchSuffix
+	lib:libKF6Solid$secondaryArchSuffix
+	lib:libKF6SyntaxHighlighting$secondaryArchSuffix
+	lib:libKF6TextEditor$secondaryArchSuffix
+	lib:libKF6WidgetsAddons$secondaryArchSuffix
+	lib:libKF6XmlGui$secondaryArchSuffix
+	# Qt6
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Network$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:libQt6Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libkomparediff2$secondaryArchSuffix >= 6
+	# KF6
+	extra_cmake_modules$secondaryArchSuffix
+	devel:libKF6Codecs$secondaryArchSuffix
+	devel:libKF6ConfigCore$secondaryArchSuffix
+	devel:libKF6CoreAddons$secondaryArchSuffix
+	devel:libKF6DocTools$secondaryArchSuffix
+	devel:libKF6GuiAddons$secondaryArchSuffix
+	devel:libKF6I18n$secondaryArchSuffix
+	devel:libKF6IconThemes$secondaryArchSuffix
+	devel:libKF6JobWidgets$secondaryArchSuffix
+	devel:libKF6Parts$secondaryArchSuffix
+	devel:libKF6TextEditor$secondaryArchSuffix
+	devel:libKF6WidgetsAddons$secondaryArchSuffix
+	devel:libKF6XmlGui$secondaryArchSuffix
+	# Qt6
+	devel:libQt6Core$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt$secondaryArchSuffix
+	cmd:msgmerge$secondaryArchSuffix
+	"
+
+PATCH()
+{
+	# disable docs generation
+	sed -e '/add_subdirectory(doc)/ s/^#*/#/' -i CMakeLists.txt
+	sed -e '/kdoctools_install(po)/ s/^#*/#/' -i CMakeLists.txt
+}
+
+BUILD()
+{
+	sed -e '/add_subdirectory(doc)/ s/^#*/#/' -i CMakeLists.txt
+	sed -e '/kdoctools_install(po)/ s/^#*/#/' -i CMakeLists.txt
```

Comment:
This is already done in `PATCH()` above.

Reply:
lol, forgot to remove these lines :) 👍